rotary/rules/assembly.smk

```diff
@@ -104,9 +104,20 @@ rule finalize_assembly:
         os.symlink(source_relpath,str(output.info))
 
 
+rule add_circular_info_to_annotation_stats:
+    input:
+        "{sample}/assembly/{sample}_circular_info.tsv"
+    output:
+        "{sample}/stats/{sample}_circular_info.tsv"
+    shell:
+        """
+        cp {input} {output}
+        """
+
 rule assembly:
     input:
         expand("{sample}/assembly/{sample}_assembly.fasta",sample=SAMPLE_NAMES),
-        expand("{sample}/assembly/{sample}_circular_info.tsv", sample=SAMPLE_NAMES)
+        expand("{sample}/assembly/{sample}_circular_info.tsv", sample=SAMPLE_NAMES),
+        expand("{sample}/stats/{sample}_circular_info.tsv", sample=SAMPLE_NAMES)
```

Comment:
If the above reference is changed to `"{sample}/assembly/{sample}_circular_info.tsv"`, then this input should be changed as well (and moved from `rule assembly` to `rule polish`. What do you think?